test-cases/longevity/longevity-cdc-100gb-8h-multi-dc-topology-changes.yaml

```diff
@@ -0,0 +1,27 @@
+test_duration: 500
+
+stress_cmd: [ "cassandra-stress user no-warmup profile=/tmp/cdc_profile_multidc.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,delete1=1)' cl=QUORUM duration=480m -port jmx=6868 -mode cql3 native -rate threads=75",
+              "cassandra-stress user no-warmup profile=/tmp/cdc_profile_multidc_preimage.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,delete1=1)' cl=QUORUM duration=480m -port jmx=6868 -mode cql3 native -rate threads=75",
+              "cassandra-stress user no-warmup profile=/tmp/cdc_profile_multidc_postimage.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,delete1=1)' cl=QUORUM duration=480m -port jmx=6868 -mode cql3 native -rate threads=75",
+              "cassandra-stress user no-warmup profile=/tmp/cdc_profile_multidc_preimage_postimage.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,delete1=1)' cl=QUORUM duration=480m -port jmx=6868 -mode cql3 native -rate threads=75"
+             ]
+
+n_db_nodes: '4 4'
+instance_type_db: 'i3.4xlarge'
```

Comment:
maybe 2xlarge is enough?
Is the load high?

Reply:
load is high, and also we have at 4 table with cdc enabled. Smaller instances could be not enough